### PR TITLE
cxl jwplayer upgrades & tweaks

### PR DESCRIPTION
**Pull request by saas786**

### CODE REVIEW

**Comment by lkraav** [comment]
Is there a Before / After screencast you could record for this? Or side-by-side even.

File: packages/cxl-ui/src/components/cxl-jw-player/mixins/NextUpMixin.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import { html, render } from 'lit';
+import style from '../../../styles/cxl-jw-player/cxl-jw-player-next-up-css';
+export function NextUpMixin(BaseClass) {
+  class Mixin extends BaseClass {
+    __nextUpCTA;
```
Comment (lkraav) [inline packages/cxl-ui/src/components/cxl-jw-player/mixins/NextUpMixin.js:5]: If we look at Lit, https://lit.dev/docs/api/decorators/ shows all "internal" stuff is single-underscore. I think we should follow upstream for such.

Ideally there could be an ESLint rule to auto-fix.

File: packages/cxl-ui/scss/cxl-jw-player/cxl-jw-player-next-up.scss [hunk 2]
```diff
@@ -1,3 +1,9 @@
...
-  pointer-events: all;
+cxl-jw-player {
+  .jw-nextup {
+    display: none;
+  }
+
+  .jw-next-up-cta {
+    pointer-events: all;
+  }
 }
```
Comment (lkraav) [inline packages/cxl-ui/scss/cxl-jw-player/cxl-jw-player-next-up.scss:9]: But this should be a global style, if not using `::slotted()`?

File: packages/cxl-ui/src/components/cxl-jw-player/mixins/NextUpMixin.js (hunk 1 above)
Comment (anoblet) [reply packages/cxl-ui/src/components/cxl-jw-player/mixins/NextUpMixin.js:5]: I believe double underscore represents private methods and properties while single underscore represents private methods and properties.

Lit does use private properties internally (https://github.com/lit/lit/blob/b0c3f82ef0f97326a205e77e7e1043b75a5cc53f/packages/lit-html/src/lit-html.ts#L651).

Unless I'm certain I'm certain a subclass should be able to override a method or property, I tend to opt for private.

I don't mind switching to protected, and I can start with this mixin. Refactoring the rest of the component will take some time.

We were trying to ship this component soon for limited use on the dashboard for A/B testing.

File: packages/cxl-ui/scss/cxl-jw-player/cxl-jw-player-next-up.scss (hunk 2 above)
Comment (anoblet) [reply packages/cxl-ui/scss/cxl-jw-player/cxl-jw-player-next-up.scss]: It's a peculiar case. Ideally everything would exists in the shadow dom, though JW Player doesn't like it. The player itself is in the light dom while everything else is in the shadow dom.

In this case you are correct. I have moved these styles.

File: packages/cxl-ui/src/components/cxl-jw-player/mixins/NextUpMixin.js (hunk 1 above)
Comment (anoblet) [reply packages/cxl-ui/src/components/cxl-jw-player/mixins/NextUpMixin.js]: I've refactored all methods and properties to protected.

**Comment by anoblet** [comment]
Would someone be able to take a look at this next week? It would be nice to use `cxl-jw-player` instead of Vimeo in the `cxl-app-layout-2c-r` story. This would help when tackling this issue: https://app.clickup.com/t/861mb94pw

**Comment by pawelkmpt** [comment]
@lkraav

**Comment by anoblet** [comment]
Closed in favor of: https://github.com/conversionxl/aybolit/pull/433

**Pull request closed without merging**